ui-guide/Components/Modals.purs

```diff
@@ -14,18 +14,22 @@ import Halogen.HTML.Properties as HP
 import Ocelot.Block.Button as Button
 import Ocelot.Block.Card as Card
 import Ocelot.Block.FormField as FormField
-import Ocelot.Block.Modal as Modal
 import Ocelot.Block.Format as Format
-import Ocelot.Components.Typeahead.Input as TA
 import Ocelot.Components.Typeahead as TACore
+import Ocelot.Components.Typeahead.Input as TA
 import Ocelot.HTML.Properties (css)
-import UIGuide.Utilities.Async as Async
+import Ocelot.Partials.Modal as Modal
```

Comment:
This is a general complaint, not one specific to your PR, but we should choose a singular vs plural convention and stick to it.
```purescript
Ocelot.Block.X
Ocelot.Component.X
Ocelot.Partial.X
UIGuide.Utility.X
```